extensions/web3-profile/src/search-web3-profile.tsx

```diff
@@ -25,6 +25,7 @@ export default function Command() {
   return (
     <List isLoading={isLoading} searchBarPlaceholder="ENS search" onSearchTextChange={setSearchTerm} throttle>
       <List.EmptyView
+        title={searchTerm.length ? "No Results" : "Search For ENS Name"}
```

Comment:
ahh yeah, thanks for that. totally forgot 😄

Reply:
Could we do something about the logo here? Can you make a logo in `64x64px` instead? I don't think we need the text here 🙂

Reply:
to be honest, i like it how it is! the overflowed text on the logo gives it a bit of character and sets a nice vibe imo

but if this is breaking any strict guidelines, happy to change it 👍